bin/echo.c

```diff
@@ -53,7 +53,12 @@ const char *sig_hash_strs[] = {
 
 void print_s2n_error(const char *app_error)
 {
-    fprintf(stderr, "[%d] %s: '%s' : '%s'\n", getpid(), app_error, s2n_strerror(s2n_errno, "EN"),
+    fprintf(stderr,
+            "[%d] %s: '%s' (%s) : '%s'\n",
```

Comment:
Nit: this is a pretty complex format string-- maybe include an example output above?